src/controllers/index.js

```diff
@@ -309,8 +309,12 @@ Controllers.manifest = function (req, res) {
 			density: 4.0,
 		});
 	}
-
-	res.status(200).json(manifest);
+	plugins.fireHook('filter:manifest.build', manifest, function (err, manifest) {
```

Comment:
One last request. Can you change the data passed to this hook to an object like below?

```
plugins.fireHook('filter:manifest.build', { req: req, res: res, manifest: manifest }, function (err, data) {
    if (err) {
	return next(err);
    }
    res.status(200).json(data.manifest);
});
```

Reply:
It was actually my first idea, but I noticed that some other hooks pass objects without wrapping them in other objects, so because `manifest` is an object i decided to go that route.
But I just didn't think about passing `req` and `res` too, and with that it makes more sense to do it this way :) 
Thanks, fixed.

Reply:
Yeah it also matches our other `build` hooks.